src/data/set/lattice.lean

```diff
@@ -358,6 +358,10 @@ theorem bUnion_union (s t : set α) (u : α → set β) :
   (⋃ x ∈ s ∪ t, u x) = (⋃ x ∈ s, u x) ∪ (⋃ x ∈ t, u x) :=
 supr_union
 
+lemma Union_subtype {α β : Type*} (s : set α) (f : α → set β) :
```

Comment:
```suggestion
@[simp] lemma Union_subtype {α β : Type*} (s : set α) (f : α → set β) :
```
Let's try Eric's suggestion. The linter seems happy enough with it (at least in this file).

If it builds, feel free to merge. Thanks!
bors d+